python-client/trustedanalytics/core/frame.py

```diff
@@ -1308,6 +1312,10 @@ class VertexFrame(_DocStubsVertexFrame, _BaseFrame):
     _entity_type = 'frame:vertex'
 
     @api
+    @arg('source','?',"""?""")
```

Comment:
Hmm... you're highlighting an issue we've got.  VertexFrame (and EdgeFrame) constructors are not really meant to be part of the public API --i.e. users shouldn't create these frames manually.  Instead, these frames are created with the graph API. 

The documentation for the __init__ method for these objects should just reference the Graph API.

Also, in general, do not document private arguments that start with underscores,  like "_info"  --thanks